Source/EasyNetQ/PullingConsumer.cs

```diff
@@ -359,6 +359,7 @@ public async Task<PullResult> PullAsync(CancellationToken cancellationToken = de
                 basicGetResult.Body
             );
             var interceptedMessage = interceptor.OnConsume(message);
+            interceptor.OnConsumed(interceptedMessage);
```

Comment:
Please pay attention for that line. In essence, both `OnConsume/OnConsumed` calls are formal here since message was already "consumed".